cmd/cluster-driver-registrar/k8s_register.go

```diff
@@ -79,6 +69,14 @@ func kubernetesRegister(
 	}
 }
 
+func cleanup(clientSet *k8scsiclient.Clientset, csiDriver *k8scsi.CSIDriver) {
+	c := make(chan os.Signal, 1)
```

Comment:
This introduces a tiny race condition where the signal may not be setup before the process exits

Reply:
@msau42  I moved the cleanup logic to a new function, will that introduces the race condition?

Reply:
@msau42 yes it may introduce race , updated code PTAL